ui/src/app/shared/services/projects-service.ts

```diff
@@ -103,6 +103,10 @@ export class ProjectsService {
         return requests.get(`/projects/${name}`).then(res => res.body as models.Project);
     }
 
+    public getGlobalProjects(name: string): Promise<models.GlobalProjectState> {
+        return requests.get(`/projects/${name}/globalprojects`).then(res => res.body as models.GlobalProjectState);
```

Comment:
I think it not worth defining a separate interface for `GlobalProjectState` . Please remove GlobalProjectState and just use:

```
public getGlobalProjects(name: string): Promise<models.Project[]> {
        return requests.get(`/projects/${name}/globalprojects`).then(res => res.body.globalprojects as models.Project[]);
}
```